build.gradle

```diff
@@ -26,7 +26,8 @@ buildscript {
 
     dependencies {
         classpath("org.springframework.boot:spring-boot-gradle-plugin:1.3.5.RELEASE")
-        classpath "gradle.plugin.com.contrastsecurity:ContrastGradlePlugin:1.0-SNAPSHOT"
+        classpath "gradle.plugin.com.contrastsecurity:ContrastGradlePlugin:1.1.1"
+        classpath "io.spring.gradle:dependency-management-plugin:0.6.1.RELEASE"
```

Comment:
You need "classpath "io.spring.gradle:dependency-management-plugin:0.6.1.RELEASE" to work with the new spring-boot-gradle-plugin" I believe.